lib/http-bridge/test/integration/Main.hs

```diff
@@ -71,13 +77,17 @@ main = hspec $ do
     describe "Cardano.WalletSpec" Wallet.spec
     describe "Cardano.Wallet.HttpBridge.NetworkSpec" HttpBridge.spec
     describe "CLI commands not requiring bridge" MnemonicsCLI.spec
-    beforeAll startCluster $ afterAll killCluster $ after tearDown $ do
+    beforeAll (startCluster ListenOnRandomPort) $ afterAll killCluster $ after tearDown $ do
         describe "Wallets API endpoint tests" Wallets.spec
         describe "Transactions API endpoint tests" Transactions.spec
         describe "Addresses API endpoint tests" Addresses.spec
         describe "Wallets CLI tests" WalletsCLI.spec
         describe "Transactions CLI tests" TransactionsCLI.spec
         describe "Addresses CLI tests" AddressesCLI.spec
+
+    let defaultPort = 8090
+    beforeAll (startCluster (ListenOnPort defaultPort)) $ afterAll killCluster $ after tearDown $ do
+        describe "Port tests using wallets CLI" (WalletsCLI.specPorts defaultPort)
```

Comment:
What's the rationale for making this one completely separated ? It's quite heavy to start a whole new cluster. Perhaps what we really need here is simply to start another wallet server listening on given port. That can be tested without requiring a new cluster.

Reply:
+1. That'd also, sort of, test the case of two wallet's attached to a single server.